cmd/client/txgen/main.go

```diff
@@ -38,6 +37,16 @@ var (
 	stateMutex sync.Mutex
 )
 
+const (
+	checkFrequency = 20 //checkfrequency checks whether the transaction generator is ready to send the next batch of transactions.
```

Comment:
Every 20 seconds it sends one batch of transactions?

Reply:
Yes.

Reply:
This is the wrong design. My original design is to make it synchronized with the block creation. Once a new block is created and received, txgen generate new txs. We shouldn't do regression on this.

Reply:
Addressed with new commit.